postgres/manage_users.py

```diff
@@ -4,21 +4,27 @@
 from botocore.exceptions import ClientError
 
 
-def drop_user(cursor, username):
+def create_or_update_db_user(conn, cursor, username, password, permissions):
     cursor.execute(f"SELECT * FROM pg_catalog.pg_user WHERE usename = '{username}'")
 
     if cursor.fetchone() is not None:
-        cursor.execute(f"DROP OWNED BY {username}")
-        cursor.execute(f"DROP USER {username}")
+      update_db_user_password(conn, cursor, username, password)  
+    else:
+      create_db_user(conn, cursor, username, password, permissions) 
 
 
-def create_db_user(conn, cursor, username, password, permissions):
-    drop_user(cursor, username)
+def update_db_user_password(conn, cursor, username, password):
+    cursor.execute(f"ALTER USER {username} WITH ENCRYPTED PASSWORD '%s'" % password)
+    conn.commit()
+    
 
+def create_db_user(conn, cursor, username, password, permissions):
+    MASTER_USERNAME = "postgres"
```

Comment:
Just a suggestion but I think `MAIN_USERNAME` rather than `MASTER_USERNAME` would read better, and maybe the value as `"postgres_user"` (if it doesn't affect anything else) I find it confusing so many places that have "postgres" as the value for the username and database name

Reply:
It is confusing, but I'm not sure we should change this in this PR - it would possibly break things for existing projects using "postgres".

Reply:
`postgres` is the name of the master user as provisioned by the terraform postgres module.  I don't think we can change this at the moment at least.  I took 'master' as the term used by aws for this user.